meta-mender-core/recipes-core/initrdscripts/files/init-install-efi-mender.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/sh -e
+#
+# Copyright (c) 2012, Intel Corporation.
+# All rights reserved.
```

Comment:
Where does this file come from? This isn't a file we can redistribute according to this license text. Though if it comes from Yocto, it's pretty likely they've made a mistake.

Reply:
I just copied the original one. But it has way too much logic in it so I have a new version coming that just starts from scratch and uses the MENDER_STORAGE_DEVICE variable instead of all the scanning that is done in the original.

Reply:
:+1: